.github/workflows/bounty-payout.yml

```
@@ -0,0 +1,109 @@
+# .github/workflows/bounty-payout.yml
+name: Bounty Payout
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  process-bounty:
+    permissions:
+        contents: read
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)